app/assets/stylesheets/locals/search_form.css.scss

```diff
@@ -1,21 +1,69 @@
-input {
-    background-color: $almost-white;
-    color: #000;
+// Default style for search forms
+.search_form {
+
+  // Default width for anything less than large screen
+  width: 220px;
+  
+  // Wider on wider screens 
+  @media (min-width: $screen-lg-min) {
+      width: 300px;
+  }
+
+  form {
+    width: 100%;
+    height: auto !important;
+    input {
+      background-color: $almost-white;
+      color: #000;
+      width: 80% !important;
+      float: left;
+      &[type="submit"] {
+          // !important to make sure it applies on search results page.
+          background-image: url(/icon-search.png) !important;
+          background-repeat: no-repeat !important;
+          background-size: contain !important;
+          background-color: inherit !important;
+          border: none !important;
+          width: 15% !important;
+          margin-left: 5px;
+      }
+    }  
+  } 
+}
+
+
+// Override search_form width in .call_to_action .well
+.call_to_action {
+  .well {
+    .search_form {
```

Comment:
style question: Do we prefer class chains like this broken out rather on a single line?